scripts/qemu/ia32-generic-ata.sh

```diff
@@ -8,8 +8,8 @@
 
 # Add ATA drives
 while [ "$#" -gt 0 ]; do
-	DRIVES+=" -ata $1"
+	DRIVES+=("-ata" "$1")
```

Comment:
Maybe it would be a good idea to declare `DRIVERS` as an array.
```
# Add ATA drives
DRIVERS=()
while [ "$#" -gt 0 ]; do
```

Reply:
Added in next commit.